open-apparel-registry/open-apparel-registry
# Increase the max length of FacilityListItem.geocoded_address

## Overview

We were running into crashes in the geocoding process because we were trying to insert more than 200 characters into the `geocoded_address` field. The value of this field comes from the `formatted_address` field in the Google geocoder response. We could not find any documentation mentioning how long this field might be so we took a guess and made the field 5x larger that it was.

Connects #342 
Connects #292

## Testing Instructions

I was not able to reproduce the issue with 14.csv as reported in #292. I was able to reproduce it with 
[handm-long-geocode-test.csv.txt](https://github.com/open-apparel-registry/open-apparel-registry/files/2990307/handm-long-geocode-test.csv.txt)


* On `develop`, upload [handm-long-geocode-test.csv.txt](https://github.com/open-apparel-registry/open-apparel-registry/files/2990307/handm-long-geocode-test.csv.txt) and run it through `parse` and `geocode` Verify that an exception is raised.
* Checkout this branch and `./scripts/manage migrate`
* Run the `geocode` process again and verify that it completes without error (some items may "fail" because they were previously geocoded successfully)
